migration/migration.go

```diff
@@ -237,8 +238,14 @@ func uploadResources(config UploadBinariesConfig) error {
 		if err != nil {
 			return errors.Trace(err)
 		}
-		// TODO(menn0) - charmstore revision
-		// TODO(menn0) - unit revisions
+		for unitName, unitRev := range res.UnitRevisions {
```

Comment:
Thinking about it though, perhaps these should be done in bulk. What do you think?